### ISSUE TITLE
Constant buffering when syncing with friends across country

### ISSUE THREAD

**Issue by ROODAY** [synclounge/synclounge#594]
### Web App
You can remove this section if your issue isn't related to the Web App

**System Details**

- Operating System: Windows 10
- Web Browser name and version: Chrome 89.0.4389.90

**Plex Media Server details**

- Version: 1.22.1.4228-724c56e62 amd64
- Operating System (Windows/Linux/Mac):  Ubuntu 20.04.2 LTS
- Running in Docker? (Yes/No): No
- Location (Local/Remote): Remote
- Remote Access Enabled (Yes/No)?: Yes

**Plex Client details**

- Name: Synclounge client
- Version: Docker Image ID 64e3ec3de671
- Network connection (Wired/Wifi): Both
- Platform: Web

### Self-Hosting
You can remove this section if your issue isn't related to Self-Hosting

**System Details**

- Operating System:  Ubuntu 20.04.2 LTS

**SyncLounge Details**

- Running in Docker? (Yes/No): Yes
- Using a Reverse Proxy (Yes/No): Yes
    If Yes, which Reverse Proxy (Nginx, Apache, Caddy, etc.): Nginx

### Issue information

**To Reproduce**

- Have a plex server that also runs synclounge located far away from you geographically
- Try using synclounge with friends located close to the server

**Describe your bug/issue**
I have a Plex Media Server located on the east coast, but I moved to the west coast recently. I still use it to watch media with friends, but for some reason only I have horrible buffering issues. We'll all wait 5 minutes for it buffer some small amount only for the buffer to run out again in 5 minutes, so it's pretty much unwatchable. If I watch stuff by myself I don't notice this issue, so not sure why via synclounge (also happens in Plex Watch Together for some shows) this happens. I tested ping to my server and it's only 100 ms, which is higher than ideal but shouldn't be causing this much buffering, right?

**Comment by ctindel** [comment]
I have seen this exact issue when someone on the west coast joins and has bad internet or something. Even though synclounge allows people to be +/- 5 seconds of each other for some reason its syncing protocol straight up breaks down when there's a misbehaving client, I don't know why. Best option might be for you to run a media server locally with the same file and tell synclounge to use that server instead of the one on the east coast.

**Comment by ROODAY** [comment]
Is that possible, essentially running multiple servers as a CDN? I thought synclounge instances would use the same file from the same server.

**Comment by ctindel** [comment]
Yes its possible, as a synclounge user you can go into settings and block the east coast server, as long as it can find another plex server in your account with the same file it will use that one instead, which could even be on the same PC that you're running the browser in.
